src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -126,7 +126,11 @@ object DepositHandler {
   }
 
   private def extractBag(mimeType: String)(implicit settings: Settings, id: String): Try[File] = {
-    def extract(file: File, outputPath: String): Unit = new ZipFile(file.getPath).extractAll(outputPath)
+    def extract(file: File, outputPath: String): Unit = {
+      val zip = new ZipFile(file.getPath)
```

Comment:
So, you are creating an anonymous sub-class of `ZipFile`, which set this to UTF-8? Looks better indeed, also nice to use `StandardCharsets`!